composer.json

```diff
@@ -16,22 +16,22 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.2",
+		"php": ">=5.6",
 		"composer/installers": "*",
-		"silverstripe/framework": "^3.1",
-		"guzzle/guzzle": "^3.9",
+		"silverstripe/framework": "4.0.x-dev",
+		"guzzlehttp/guzzle": "~3.8.1",
 		"symfony/event-dispatcher": "^2.6.0@stable",
 		"symfony/http-foundation": "^2.6.0"
 	},
 	"require-dev": {
-		"phpunit/phpunit": "^3.7"
+		"phpunit/phpunit": "~5.0"
 	},
 	"suggest": {
 		"ext-fileinfo": "Improved support for file mime detection"
 	},
 	"extra": {
 		"branch-alias": {
-			"dev-master": "2.1.x-dev"
+			"dev-master": "3.0.x-dev"
```

Comment:
Let's use `3.x-dev` please, then we won't need to change it when 3.0.0 stabilises

Reply:
Updated in the master branch, rebase to get it